packages/ckeditor5-language/src/textfragmentlanguage.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module language/textfragmentlanguage
+ */
+
+import { Plugin } from 'ckeditor5/src/core';
+
+import TextFragmentLanguageEditing from './textfragmentlanguageediting';
+import TextFragmentLanguageUI from './textfragmentlanguageui';
+
+/**
+ * The text fragment language plugin.
+ *
+ * This feature allows setting language of parts of the content using simple dropdown button. For example,
+ * use it when you would like to indicate that the specific quote is in different language than the rest of the content.
+ * Refer to [WCAG 3.1.2 Language of Parts](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-other-lang-id.html) specification
+ * to learn more.
+ *
+ * To change UI editor language, refer to [Setting the UI language](https://ckeditor.com/docs/ckeditor5/latest/features/ui-language.html)
+ * guide.
+ *
+ * For more information about this feature check the {@glink api/textfragmentlanguage package page}.
+ *
+ * This is a "glue" plugin which loads the {@link module:language/textfragmentlanguageediting~TextFragmentLanguageEditing
+ * text fragment language editing feature} and
+ * {@link module:language/textfragmentlanguageui~TextFragmentLanguageUI text fragment language UI feature}.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class TextFragmentLanguage extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ TextFragmentLanguageEditing, TextFragmentLanguageUI ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'TextFragmentLanguage';
+	}
+}
+
+/**
+ * The available {@link module:language/textfragmentlanguage} options allowing setting language of parts of the content.
```

Comment:
I went with `language.textFragmentLanguage` option name to match the plugin name, but it would be probably nicer to simply name it `language.textFragment` to avoid duplication. WDYT?

Reply:
After a second thought, it probably makes a bit more sense to keep a longer format, so the plugin name. Otherwise, it will be hard to tell what feature is actually introducing this option which may affect discoverability.